README.md

```diff
@@ -43,6 +43,35 @@ reminderbatch -w <WAVE_NUMBER> -b <STARTING_BATCH_NUMBER> -a <ACTION_PLAN_ID> --
 reminderbatch -w <WAVE_NUMBER> -b <STARTING_BATCH_NUMBER> -a <ACTION_PLAN_ID> --insert-rules --trigger-date-time=<DATE_TIME> --max-cases=1000000 
 ```
 
+## Response-driven reminder scheduling
+We can create response-driven action rules from a provided CSV file of LSOAs from a bucket. You will need to copy this file from the bucket to your cloud project, then copy to the running toolbox instance.
+
+#### Get counts of cases for response-driven reminders
+To get a count of cases using the LSOAs file:
```

Comment:
Can the README include an example or description of the format of the LSOA file?

Reply:
I was holding off from giving file example as the file spec doc was subject to change. Wouldn't hurt to put something in here though even if it needs to change later 👍